src/__test__/NewCountries.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
+import NewCountries from '../components/Countries/NewCountries';
+// eslint-disable-next-line
+import store from '../redux/configureStore';
+
+it('should render the NewCountry component Correctly', () => {
+  const newCountries = render(
+    <BrowserRouter>
+      <Provider store={store}>
+        <NewCountries />
+      </Provider>
+    </BrowserRouter>,
+  );
+  expect(newCountries).toMatchSnapshot();
+});
```

Comment:
- [ ] This test is failing, please checkout the screenshot below and kindly fix the issue.  

![failing_test](https://user-images.githubusercontent.com/65852621/218037001-161f5891-3286-4317-895f-7c5e21a4f91f.PNG)